libsplinter/src/admin/service/shared.rs

```diff
@@ -664,6 +664,38 @@ impl AdminServiceShared {
                 self.propose_circuit(payload, "local".to_string())
             }
             CircuitManagementPayload_Action::CIRCUIT_PROPOSAL_VOTE => {
+                let proposal_vote = payload.get_circuit_proposal_vote();
+
+                // validate vote proposal
+                // check that the circuit proposal exists
+                let circuit_proposal = self
+                    .get_proposal(proposal_vote.get_circuit_id())
+                    .map_err(|err| {
+                        ServiceError::UnableToHandleMessage(Box::new(
+                            AdminSharedError::ValidationFailed(format!(
+                                "error occured when trying to get proposal {}",
```

Comment:
s/occured/occurred/

Reply:
Damn, I fixed that typo in one place, but reproduced it else where...

Fixed

Reply:
s/else where/elsewhere/